### ISSUE TITLE
use more standard license

### ISSUE THREAD

**Issue by fommil** [chrissimpkins/Hack#209]
Hi, great font! I've started using it :smile:

Your license https://github.com/chrissimpkins/Hack/blob/master/LICENSE.md is non-standard (according to my work's legal department) and is therefore not allowed as it is too costly to approve new ad-hoc licenses (the Bitstream license is not approved for the same reason).

Can you please use a standard licence such as CC-BY-SA, GPL or Apache 2.0?

Note that although the Hack website says that this font is Libre, there is no listing on the FSF license list http://www.gnu.org/licenses/license-list.en.html which would back up this claim.

**Comment by chrissimpkins** [comment]
Thanks for the feedback Sam.  I wish it were that simple.  Hack is a project that uses Bitstream Vera Sans Mono and DejaVu Sans Mono upstreams.  The BVSM license must always be associated with this derivative typeface and must include the reserved font names Bitstream and Vera.  The DejaVu contributions to BVSM are in the public domain.  Our Hack Open Font License includes elements from the SIL Open Font License to make several issues explicit beyond what is stated in the Bistream Vera license.  Furthermore, it declares a reserved font name "Hack" for the typeface.  

I have been through the license with representatives from SIL and am currently working through licensing issues with @davelab6 in order to try to find an acceptable approach for inclusion in Google Fonts.  This is not a simple issue.  We are straddling an older libre license, contributions that are dedicated to the public domain (which I have recently learned is an issue that is defined very differently by country), and the newer SIL style license structure that is commonly used for libre open source typefaces. 

So, the short answer is that we can't simply change it to a commonly used and widely acceptable FLOSS license to make it simple for the lawyers.  I really apologize that this has limited your use of the fonts.

Having said that, our license does support the [four freedoms of Free software](https://www.gnu.org/philosophy/free-sw.html).  What is the use case for which your legal group feels that these fonts are "not allowed" and what are their concerns?

**Comment by fommil** [comment]
Its just time and effort. If it's not on the accepted list they need a business case to review it. Probably the same at all large corporations.

**Comment by chrissimpkins** [comment]
To install on boxes at work, to embed in applications? what are you doing with the fonts?

**Comment by fommil** [comment]
to install, just to use it.

**Comment by chrissimpkins** [comment]
wow ok...

**Comment by fommil** [comment]
Well it is the primary use of a license: an agreement between the licensor (author) and recipient (user).

Every piece of software that is used on a corporate desktop needs to have the license approved by legal, and if the license isn't a standard one then you need to spend internal funds to buy some of the legal department's time to review the license (even if it is trivial).

In most companies, downloading / installing software without going through appropriate channels is a sackable offense, and when it comes to installing fonts on Windows it requires interaction with the Windows admins because it isn't something a mere mortal user can do.

**Comment by fommil** [comment]
btw, maybe I misunderstand the license. But isn't "Neither the Font Software nor any of its individual components, in original or modified versions, may be sold by itself." in conflict with the free/libre right to charge for original or modified versions?

**Comment by chrissimpkins** [comment]
This must be present as it exists in the Bitstream Vera license and it is part of the language of the SIL OFL license which is part of the FSF libre font list (and from which our own license addendum is derived).  Here is FSF's take on it:

>The Open Font License (including its original release, version 1.0) is a free copyleft license for fonts. Its only unusual requirement is that when selling the font, you must redistribute it bundled with some software, rather than alone. Since a simple Hello World program will satisfy the requirement, it is harmless. Neither we nor SIL recommend the use of this license for anything other than fonts.

**Comment by fommil** [comment]
Cool, thanks for clarifying the origins! Would be good if the FSF had a section for the Bitstream Vera licence, but that's their responsibility.

**Comment by chrissimpkins** [comment]
Again, sorry that this quagmire limits your use.  There is absolutely no intent to do that.

**Comment by davelab6** [comment]
> Again, sorry that this quagmire limits your use. There is absolutely no intent to do that.

I agree. I hope we can make an OFL release in the future. Thanks for your patience :)

**Comment by bowlofeggs** [comment]
I've started a thread about whether this font can be included in Fedora:

https://lists.fedoraproject.org/archives/list/legal@lists.fedoraproject.org/thread/MJ4Q3D6JLDVX4OHSF3YXY76Z6BYSKKQR/

**Comment by spotrh** [comment]
To chime in here, I went ahead and allowed fonts under the Hack Open Font License into Fedora, because it is as "free" as the SIL OFL and Bitstream Vera licenses. That said, I strongly encourage you to get off it. :)

**Comment by davelab6** [comment]
@bowlofeggs I admire your persistence!! :)

**Comment by bowlofeggs** [comment]
Haha, thanks @davelab6!

**Comment by chrissimpkins** [comment]
@bowlofeggs ty! Shawn Starr reached out to me to work on a build chain for the fonts so that they can be released on Fedora.  Let's see what we can do.

**Comment by chrissimpkins** [comment]
@spotrh :+1: thank you very much for your help!

**Comment by spstarr** [comment]
From Fedora packaging folks "There is no absolute requirement that fonts be built from source" however this is desirable if we can do it.

Maybe we can do this in phases, for Phase I, ship RPM without building from source, for Phase II, work with Chris on seeing how to build the fonts from source?

**Comment by spotrh** [comment]
Is it worth keeping this open until you can do an OFL release?

**Comment by chrissimpkins** [comment]
@spotrh There will never be a pure OFL release of Hack because the upstream is Bitstream Vera Sans Mono.  The Bitstream license must stay with the fonts.  I have discussed this issue at length with @davelab6 and the SIL team.  From what I can gather we unfortunately reached too deep into the history of font licensing with our upstream and are stuck unless whoever now claims ownership of the Bitstream license (Bitstream no longer exists) is willing to modify their license...  

Here is the Wikipedia history of Bitstream:

> Bitstream Inc. was a type foundry that produced digital typefaces. Founded in 1981 by Matthew Carter and Mike Parker among others, it claims to be the oldest such company. It was located in Marlborough, Massachusetts. The font business, including MyFonts, was acquired by Monotype Imaging in March 2012.[2][3] The remainder of the business, responsible for Pageflex and Bolt Browser, was spun off to a new entity named Marlborough Software Development Holdings Inc.[4][5] It was later renamed Pageflex, Inc following a successful management buyout in December 2013.[6]

I tried to contact Monotype to discuss the licensure issue over a year ago and did not get anywhere with it.

There isn't much that we can do about it.  Hack started because I loved BVSM.  To re-draw the glyphs away from that original design would eliminate the character of this much loved typeface lineage which is its attraction for most users.

I am more than willing to reopen if there is an interest in ongoing licensing discussion in this thread, but for now this is where we are.
